src/aaudio/AudioStreamAAudio.h

```diff
@@ -117,9 +117,12 @@ class AudioStreamAAudio : public AudioStream {
      */
     void launchStopThread();
 
-    // Time to sleep in order to prevent a race condition with a callback after a close().
-    // Two milliseconds may be enough but 10 msec is even safer.
-    static constexpr int kDelayBeforeCloseMillis = 10;
+public:
+    int32_t getMDelayBeforeCloseMillis() const;
```

Comment:
Should we make this uint32_t?

Reply:
The [Google C++ style guide](go/cstyle#Integer_Types) says:

> You should not use the unsigned integer types such as uint32_t, unless there is a valid reason
> such as representing a bit pattern rather than a number, or you need defined overflow
> modulo 2^N. In particular, do not use unsigned types to say a number will never be negative.
> Instead, use [assertions](https://engdoc.corp.google.com/eng/doc/devguide/cpp/primer#google_assert) for this.